Panic: Unauthorized when Telegram Bot Token is faulty

```
➜  immo_trakt git:(main) ✗ go run main.go
go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
go: downloading github.com/go-telegram-bot-api/telegram-bot-api v4.6.4+incompatible
go: downloading github.com/go-co-op/gocron v1.5.0
go: downloading github.com/kelseyhightower/envconfig v1.4.0
go: downloading golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
go: downloading github.com/robfig/cron/v3 v3.0.1
go: downloading github.com/technoweenie/multipartstreamer v1.0.1
2022/03/24 18:05:14 ImmoTrakt is going to run with following configuration: 
Frequency: 1m
Include Existing Offers: false
Search: https://www.immobilienscout24.de/Suche/de/berlin/berlin/wohnung-mieten?price=-950.0&pricetype=calculatedtotalrent&enteredFrom=result_list
Exclude WBS: false
Exclude Tausch: false
2022/03/24 18:05:14 Unauthorized
panic: Unauthorized

goroutine 1 [running]:
log.Panic({0xc000177f10, 0x2f, 0xc00007d530})
	/usr/lib/go/src/log/log.go:354 +0x65
main.main()
	/home/a0n/immo_trakt/main.go:102 +0x219
exit status 2
```

hey! I keep running into the same issue and also tried putting the token bot, user id and the search link in quotation marks. Do you have an idea how I could solve the issue?

```Fabians-MBP-2:immo_trakt fabian$ go run main.go
2022/04/06 12:23:29 ImmoTrakt is going to run with following configuration: 
Frequency: 1m
Include Existing Offers: false
Search: https://www.immobilienscout24.de/Suche/de/berlin/berlin/wohnung-mieten?haspromotion=false&numberofrooms=2.0-&price=-1800.0&exclusioncriteria=projectlisting,swapflat&pricetype=rentpermonth&geocodes=110000000702,110000000801,110000000406,110000000101,110000000102,110000000103,110000000202,110000000301,110000000401,110000000104,110000000105,110000000402,110000000106,110000000601,110000000701&sorting=2&enteredFrom=result_list
Exclude WBS: false
Exclude Tausch: false
2022/04/06 12:23:29 Unauthorized
panic: Unauthorized

goroutine 1 [running]:
log.Panic({0xc00012df10?, 0x2e?, 0xc00007f530?})
	/usr/local/go/src/log/log.go:385 +0x65
main.main()
	/Users/fabian/PycharmProjects/immo_trakt/main.go:102 +0x219
exit status 2```

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,3 @@
 # Config values
 config.yml
+immo-trakt

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 # Config values
 config.yml
 immo-trakt
+

--- a/config_skeleton.yml
+++ b/config_skeleton.yml
@@ -2,10 +2,10 @@ immo_trakt:
   frequency: 1m #every 1 min
   include_existing_offers: false
 telegram:
-  token: botToken
+  token: "botToken"
   chat_id: #Can be left empty
 immobilien_scout:
-  search: https://www.immobilienscout24.de/Suche/de/berlin/berlin/wohnung-mit-balkon-mieten?price=-1000.0&pricetype=calculatedtotalrent&enteredFrom=result_list
+  search: "https://www.immobilienscout24.de/Suche/de/berlin/berlin/wohnung-mit-balkon-mieten?price=-1000.0&pricetype=calculatedtotalrent&enteredFrom=result_list"
   exclude_wbs: false #to exclude offers that contains word WBS in their title
   exclude_tausch: false #to exclude offers that contains word TAUSCH in their title
   exclude_senior: false #to exclude offers that contains word SENIOR in their title

